ptsprojects/testcase_db.py

```diff
@@ -7,7 +7,7 @@ class TestCaseTable:
     def __init__(self, name, database_file=DATABASE_FILE):
         self.database_file = database_file
         self._open()
-        self.name = name
+        self.name = name.replace(r'-', r'_')
```

Comment:
It seems that this is not related. If this solves some issue please create a separate commit and describe the problem.